pages/wishlist/wishlist.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Apex Furniture | Wishlist</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <!-- Header -->
+    <nav class="navbar">
+      <div class="logo font-weight-black align-items-center h2">
+        <a
+          class="display-flex font-weight-black align-items-center h2"
+          href="/index.html"
+        >
+          <span class="site-name">Apex &nbsp;</span><span>Furniture</span>
+        </a>
+      </div>
+      <div class="search-bar">
+        <input
+          class="search-input"
+          placeholder="What are you looking for?"
+          type="text"
+        />
+        <span class="search-icon">
+          <i class="fa fa-search"></i>
+        </span>
+      </div>
+      <div class="navigation font-weight-semibold">
+        <a class="h3" id="login" href="#"
+          ><i class="fa fa-user-circle-o"></i
+        ></a>
+        <a class="h3" href="/pages/wishlist/wishlist.html"
+          ><i class="fa fa-heart"></i
+        ></a>
+        <a class="h3" href="#"><i class="fa fa-shopping-cart"></i></a>
+      </div>
+    </nav>
+
+    <main>
+      <section class="wishlist-container">
+        <h2 class="page-title text-align-center">My Wishlist</h2>
+        <div class="wishlist-row">
+          <div class="card-vertical">
+            <div class="card-image">
+              <img
+                class="responsive-image"
+                src="/assests/images/product_image.jpg"
+                alt="product_image"
+              />
+            </div>
+            <div class="card-body pad-1 h5">
+              <h3 class="product-heading font-weight-semibold margin-bottom-1">
+                Gaming Chair
+                <span class="card-icon">
+                  <i class="fas fa-heart"></i>
+                </span>
+              </h3>
+              <p class="product-description font-weight-regular">
+                Gaming Chair Adjustable Back. Red, Black, DIY(Do-It-Yourself)
+              </p>
+              <p class="product-pricing">
+                <b>₹12000</b>
+                <span class="strike-through font-weight-regular">₹1200</span>
```

Comment:
can use p tag instead off span tags makes it more semantic

Reply:
Okay will look into it. Thanks for the suggestion